### ISSUE TITLE
Issue creating option value

### ISSUE THREAD

**Issue by jswift** [aligent/bigcommerce-v3-api-php-client#103]
What is the method to create a option value? So far I have:

 

```php
$productOptionValue = new BigCommerce\ApiV3\ResourceModels\Catalog\Product\ProductOptionValue();
// ... setup option values

$response = $api->catalog()->product($id)->options($optionid)->Values()->create($productOptionValue);
```

The issue is, it is not inserting the option ID in the url. It is attemping to post to “/catalog/products/141/options/0/values”, where 141 is the product id. The option value should be 115 not 0.

**Comment by jswift** [comment]
This test in [ProductOptionValuesApiTest](/aligent/bigcommerce-v3-api-php-client/tree/feature/product-option-values-improvements/tests/BigCommerce/Api/Catalog/Products/ProductOption/ProductOptionValuesApiTest.php) demonstrates a create call that appears to use the correct URL.

There was a minor issue which resulted in the error `Fatal error: Class BigCommerce\ApiV3\ResponseModels\OptionsResponse contains 1 abstract method and must therefore be declared abstract or implement the remaining methods`
though this didn't happen in this case.

**Comment by jswift** [comment]
This minor issue is fixed in 1.6.4
